imagetool/src/test/java/com/oracle/weblogic/imagetool/util/DockerfileBuilderTest.java

```diff
@@ -41,13 +41,13 @@ public void validateMustacheAliases() throws IOException {
 
     @Test
     public void setPackageInstaller() {
-        DockerfileOptions options = new DockerfileOptions().setPackageInstaller(Constants.YUM);
+        DockerfileOptions options = new DockerfileOptions("123").setPackageInstaller(Constants.YUM);
```

Comment:
What is option 123 ?

Reply:
Nothing.  DockerfileOptions requires a Unique ID for builds.  For these Unit tests, I just added a simple string since it wasn't doing an actual Docker build.  During actual runs, DockerfileOptions takes a UUID from the tool in uuid package.